Task7/Task7/Factories/EBookCatalogLibrary.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Task7.Entities;
+using Task7.Factories.Abstraction;
+using Task7.Repositories.Abstraction;
+
+namespace Task7.Factories
+{
```

Comment:
EBookCatalogLibrary and PaperBookCatalogLibrary can't implement  IAbstractCatalogFactory interface, these have to be concrete factories, creation classes which has to return Library with necessary context. Change names